### DISCUSSION TITLE
Creating ArchR project from existing/saved arrow files

### DISCUSSION THREAD

**Discussion by CarstenVos** [GreenleafLab/ArchR#1733] [Questions / Documentation]
I am new to ArchR and supposed to work with data from a colleague's snATAC-seq project. I was provided with an RDS file of a gene activity matrix he created in ArchR, as well as the associated arrow files.

The problem is that I have found no way to create a project from these. The tutorial/manual is great, but it focuses exclusively on a workflow in which you create the arrow files yourself, followed by creating a project that you associate them with, and I have not found any function that allows me to create a project from pre-existing arrow files. I have googled around for some sort of load function, but cannot find anything.

Several discussions on here have mentioned this is possible, but I am unable to find the syntax.
 
Any chance a helpful soul could point me in the right direction? :)

**Comment by rcorces** [comment]
This is not the approved way of sharing an ArchRProject. Your colleague should save the entire project using `saveArchRProject()` and share that entire output directory with you. Then you should load it on your end using `loadArchRProject()`. After analysis has been done, there are too many connections between Arrow Files and other project data to regenerate a project just from the Arrow Files.

"naked" Arrow Files directly output from `createArrowFiles()` and without any additional analysis done could be shared as is and then `createArchRProject()` could be used to make an ArchRProject object. But this will not work if additional information has been added to the Arrow Files.

**Reply by CarstenVos** [reply]
Thanks for your input! My colleague is abroad and gave me access to some folders here that contained the arrow files, but given I am already running into the **Error in H5Fopen(file, flags = flags, fapl = fapl, native = native) : HDF5. File accessibility. Unable to open file.** error when trying to load the arrow files, I guess I really have to ask him to somehow share the project via _saveArchRProject()_.
